### ISSUE TITLE
OpenID Connect

### ISSUE THREAD

**Issue by joshp23** [wallabag/wallabag#4205]
It would be useful to be able to enable OpenID Connect SSO functionality. A basic OpenID Connect library is available [here](https://github.com/jumbojett/OpenID-Connect-PHP) that works with self hosted OIDC Provider solutions such as KeyCloak. This could also enable so called social login with ie. guthub  credentials.

This single sign on solution is useful for users that host several apps, and want to streamline and simplify user management and authentication across apps.

**Comment by badrihippo** [comment]
I agree with this. Just to be clear, OpenID support doesn't necessarily mean using closed services like GitHub or Facebook as assumed in the old discussion at #795. One could also have their own internal setup using something like [SimpleLogin](https://docs.simplelogin.io/docs/app/) or a custom auth backend (something that I, for one, would find very useful).

**Comment by joshp23** [comment]
Still hoping for this for use with my own OIDC server.

**Comment by helmut72** [comment]
Would be great to login users to Wallabag using OIDC. Then I don't need to login to every service I use. For Wallabag client apps, a seperate app password should help to be compatible with existing apps.

**Comment by parikhparth23** [comment]
any update on this?

**Comment by linarphy** [comment]
I'm really interested in this feature too !

**Comment by hibare** [comment]
Could this be prioritized. This is the only features that is holding me back from self-hosting wallabag

**Comment by Kdecherf** [comment]
@hibare unfortunately it is not considered as a priority for now, as we need to work on the symfony 5 update

**Comment by Nimamoh** [comment]
Too bad, look forward to this feature too.

This is a good project, thank you for your work!

**Comment by GAS85** [comment]
Really need this, to use my own e.g. Nextcloud or KeyCloak as IDP.

**Comment by EHRETic** [comment]
+1 for me, I would like to integrate it with Authentik! :-)

**Comment by Scot-Survivor** [comment]
Any updates on this?

**Comment by proofrock** [comment]
Really interested

**Comment by bioshazard** [comment]
+1 using on Coolify, want to use my local Keycloak

**Comment by proofrock** [comment]
+1

**Comment by poperigby** [comment]
Please just leave a thumbs up instead of "+1". Everyone that's subscribed to this issue gets pinged when you leave a comment. Yes, I'm aware of the irony of doing just that to say this.
